### PR TITLE
[DPEDE-526] Customize columns improvements

### PR DESCRIPTION
**Pull request by josecarlosgonzalezv**
https://ctl.atlassian.net/browse/DPEDE-526

### CODE REVIEW

**Review by SergioQCL — APPROVED** [review]
BOOM

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "backstopjs": "4.5.1",
     "browser-sync": "2.29.3",
     "cssnano": "4.1.10",
-    "cypress": "12.17.3",
+    "cypress": "13.1.0",
```
Comment (josecarlosgonzalezv) [inline package.json:41]:
```suggestion
    "cypress": "12.7.4",
```

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
     "backstopjs": "4.5.1",
     "browser-sync": "2.29.3",
     "cssnano": "4.1.10",
-    "cypress": "12.17.3",
+    "cypress": "12.7.4",
```
Comment (josecarlosgonzalezv) [inline package.json:41]:
```suggestion
    "cypress": "12.17.4",
```

File: src/chi-vue/src/components/column-customization/ColumnCustomizationModalContent.tsx [hunk 3]
```diff
@@ -73,7 +73,15 @@ export default class ColumnCustomizationContent extends Vue {
         refButton = 'buttonMoveDown';
       }
 
-      const isDisabled = (icon === 'chevron-up' && !this.canMoveUp) || (icon === 'chevron-down' && !this.canMoveDown);
+      const everyColumnsSelected = (this._selectedColumns as DataTableColumn[]).every(
+        (column: DataTableColumn) => column.locked
+      );
```
Comment (SergioQCL) [inline src/chi-vue/src/components/column-customization/ColumnCustomizationModalContent.tsx:78]: I think this name can be confusing. Shouldn't it be.. everyColumnLocked?

**Review by SergioQCL — APPROVED** [review]

**Pull request merged**